.gitignore

```diff
@@ -8,3 +8,6 @@ fuzz/hfuzz_workspace
 #IntelliJ project files
 .idea
 *.iml
+
+#Vscode project files
+.vscode
```

Comment:
I notice that we have `.idea` there as well, but isn't it customary if you use an IDE that you add the IDE's files to your global gitconfig?

`~/.gitconfig`:
```
[core]
    excludesfile = ~/.gitignore-global
```

Reply:
Interesting! I was using `.git/info/exclude` for that so far. But yeah, ideally we wouldn't have that in our ignore file (Even though I was a bit scared to look into the git blame of that line xD might have been me as a CLion user after all).

Reply:
Thanks, I was not aware of this. Will change this commit to remove the .idea files instead of adding ones for vscode